addons/mail/models/discuss/discuss_channel_member.py

```diff
@@ -185,32 +186,30 @@ def unlink(self):
         self.sudo().rtc_session_ids.unlink()  # ensure unlink overrides are applied
         return super().unlink()
 
+    def _bus_channel(self):
+        return self.partner_id or self.guest_id
+
     def _notify_typing(self, is_typing):
         """ Broadcast the typing notification to channel members
             :param is_typing: (boolean) tells whether the members are typing or not
         """
-        notifications = []
         for member in self:
             formatted_member = member._discuss_channel_member_format().get(member)
             formatted_member['isTyping'] = is_typing
-            notifications.append([member.channel_id, 'discuss.channel.member/typing_status', formatted_member])
-            notifications.append([member.channel_id.uuid, 'discuss.channel.member/typing_status', formatted_member])  # notify livechat users
-        self.env['bus.bus']._sendmany(notifications)
+            member.channel_id._bus_send('discuss.channel.member/typing_status', formatted_member)
+            self.env['bus.bus']._sendone(member.channel_id.uuid, 'discuss.channel.member/typing_status', formatted_member)  # notify livechat users
```

Comment:
To be double checked but I don't think we need this anymore.

cc @tsm-odoo

Reply:
I removed it, please advise if I should put it back

Reply:
Note that I also removed it from the mock for the tests, and everything is still green :shrug: